datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -97,6 +101,22 @@ internal class QuestionnaireViewModel(state: SavedStateHandle) : ViewModel() {
 
   private val pageFlow = MutableStateFlow(questionnaire.getInitialPagination())
 
+  private val itemAnswerOptionComponentMap =
```

Comment:
```suggestion
  private val answerValueSetMap =
```

i think answer value set map is a more accurate name. because this maps the answer value set identifier to the answer options in the answer value set. also maybe add documentation.

Reply:
Done